pkg/internal/apis/config/validate.go

```diff
@@ -130,6 +140,68 @@ func (n *Node) Validate() error {
 	return nil
 }
 
+func formatPortMapping(addr net.IP, port int, protocol PortMappingProtocol) string {
+	return fmt.Sprintf("%s/%s",
+		net.JoinHostPort(addr.String(), strconv.Itoa(port)),
+		protocol,
+	)
+}
+
+func validatePortMapping(portMappingsSet sets.String, addrSet sets.String, mapping PortMapping) error {
+	wildcardAddrIPv4 := net.ParseIP("0.0.0.0")
+	wildcardAddrIPv6 := net.ParseIP("::")
+
+	addr := net.ParseIP(mapping.ListenAddress)
+	port := int(mapping.HostPort)
+	protocol := mapping.Protocol
+	ipFamily := getIPFamily(addr)
```

Comment:
having the address we don't really need the ip family, do we?

Reply:
that was based on the assumption that `::` and `0.0.0.0` are different /